Solutions/Menes.Hosting.AspNetCore/Menes/Internal/PocoOutputBuilder.cs

```diff
@@ -0,0 +1,99 @@
+﻿// <copyright file="PocoOutputBuilder.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Menes.Internal
+{
+    using System.Collections.Generic;
+
+    using Menes.Converters;
+
+    using Microsoft.Extensions.Logging;
+    using Microsoft.OpenApi.Models;
+
+    /// <summary>
+    /// Common logic for building a response for a POCO, with output validation against the
+    /// <see cref="OpenApiOperation"/> definition.
+    /// </summary>
+    /// <typeparam name="TResponse">The response type.</typeparam>
+    internal abstract class PocoOutputBuilder<TResponse> : IResponseOutputBuilder<TResponse>
+    {
+        private readonly IEnumerable<IOpenApiConverter> converters;
+        private readonly ILogger logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PocoActionResultOutputBuilder"/> class.
+        /// </summary>
+        /// <param name="converters">The open API converters to use with the builder.</param>
+        /// <param name="logger">The logger for the output builder.</param>
+        public PocoOutputBuilder(
```

Comment:
As above - shouldn't abstract class constructors be protected?

Reply:
Yes, also fixed now - thanks!